packages/compass-testserver/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@mongodb-js/compass-testserver",
+  "productName": "compass-testserver Plugin",
```

Comment:
```suggestion
  "productName": "compass-testserver",
```

Reply:
Looks like create-workspace template is adding this by mistake, productName is only "needed" for plugins

Reply:
Yeah, thanks for the pointer, removed this line :+1: